setup.py

```diff
@@ -41,8 +41,9 @@ def _check_output(command):
 
 
 def pkg_config_version_check(pkg, version):
+    pkg_config = os.environ.get('PKG_CONFIG','pkg-config')
```

Comment:
Small nit, but the spacing looks a little bit weird, try adding a space between the parameters of .get:

`pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')`

Reply:
Force pushed a version with additional spaces.